openqa-label-known-issues

```diff
@@ -134,14 +137,16 @@ investigate_issue() {
     # against even in case when autoinst-log.txt is missing the details, e.g.
     # see https://progress.opensuse.org/issues/69178
     echo "$reason" >> "$out"
-    if [[ "$curl_output" != "200" ]] && [[ "$curl_output" != "301" ]]; then
+    # With 404 assume autoinst-log.txt is missing but we have a reason var
+    # to process the job
+    echo "DEBUG $reason .. $curl_output"
+    [[ -n "$reason" ]] && [[ "$curl_output" = "404" ]] && [[ "$reason" != null ]] \
+        && has_missing_logs_but_reason_found=1
+    if [[ "$curl_output" != "200" ]] && [[ "$curl_output" != "301" ]] && [[ "$has_missing_logs_but_reason_found" ]]; then
         # if we can not even access the page it is something more critical
         handle_unreachable "$testurl" "$out"
```

Comment:
So now the `handle_unreachable` is only done when there is a reason found?

Reply:
fixed and added new test cases.

Reply:
Now `handle_unreachable` is only called if there is no `reason`?

Reply:
yes but more accurate, it is called only when there is no `reason` and `autoinst-log.txt` isnt found. Which I think it makes sense.